CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ cmake_dependent_option(
 option(USE_DOXYGEN "Build HTML cross reference with doxygen" OFF)
 option(USE_PANDOC "Build man pages and HTML reference with pandoc" ON)
 option(USE_POC "Build small, uninstalled proof-of-concept binaries" ON)
-option(USE_QRCODEGEN "Disable libqrcodegen QR code support" ON)
+option(USE_QRCODEGEN "Enable libqrcodegen QR code support" ON)
```

Comment:
i'm down with this, but change the ON to OFF in this case. the help ought be saying what the option can do to change from the defaults. it only makes sense for this to be "enable" if the default is OFF.